services/common/src/components/forms/RenderFileUpload.tsx

```diff
@@ -361,44 +362,60 @@ export const FileUpload = (props: FileUploadProps) => {
           }}
         />
       )}
-      <FilePond
-        ref={(ref) => (filepond = ref)}
-        server={server}
-        name="file"
-        beforeDropFile={props.beforeDropFile}
-        beforeAddFile={props.beforeAddFile}
-        allowRevert={props.allowRevert}
-        onremovefile={props.onRemoveFile}
-        allowMultiple={props.allowMultiple}
-        onaddfilestart={props.addFileStart}
-        allowReorder={props.allowReorder}
-        maxFileSize={props.maxFileSize}
-        // maxFiles={props.maxFiles || undefined}
-        allowFileTypeValidation={acceptedFileTypes.length > 0}
-        acceptedFileTypes={acceptedFileTypes}
-        onprocessfiles={props.onProcessFiles}
-        onprocessfileabort={props.onAbort}
-        // oninit={props.onInit}
-        labelIdle={props?.labelIdle}
-        itemInsertLocation={props?.itemInsertLocation}
-        credits={null}
-        fileValidateTypeLabelExpectedTypesMap={fileValidateTypeLabelExpectedTypesMap}
-        fileValidateTypeDetectType={(source, type) =>
-          new Promise((resolve, reject) => {
-            // If the browser can't automatically detect the file's MIME type, use the one stored in the "accepted file types" map.
-            if (!type) {
-              const exts = source.name.split(".");
-              const ext = exts && exts.length > 0 && `.${exts.pop()}`;
-              if (ext && ext in props.acceptedFileTypesMap) {
-                type = props.acceptedFileTypesMap[ext];
-              } else {
-                reject(type);
-              }
-            }
-            resolve(type);
-          })
+      <Form.Item
```

Comment:
Put this in a Form.Item because without giving a Form.Item with a name property, _it doesn't get passed in with values to onSubmit_. Put null checks on input & meta though because tests failed without them.